superset/assets/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -139,6 +143,60 @@ class FilterBox extends React.Component {
     });
   }
 
+  cascadeFilters(filter, selectedValues) {
+    const filters = Object.keys(this.props.filtersChoices);
```

Comment:
Shouldn't this be pulling from filtersFields?  I'm not sure we even have a guarantee or ordering in the filtersChoices?